features/steps/insights_results_aggregator.py

```diff
@@ -88,6 +103,20 @@ def check_version_from_aggregator(context):
         raise Exception("Improper or missing version info in {}".format(context.output))
 
 
+@then('I should see following message in service output: "{message}"')
```

Comment:
maybe this could be a common step as it might be used in other services(?)

Reply:
yup, I'll change it, sure